portfolio/src/main/webapp/script.js

```diff
@@ -22,3 +22,22 @@ function expandCollapsible(element) {
 function navtoggle() {
   document.getElementById('navmenu').classList.toggle('visible');
 }
+
+function openModal(target, index) {
```

Comment:
it will be used once I set up the fetch request in a later commit/PR. The index will be included in the URL search parameters, for the server to know which comment section to retrieve

Reply:
Thanks for clarifying!